events/delete_bulk_messages_MOD.js

```diff
@@ -0,0 +1,89 @@
+module.exports = {
+
+//---------------------------------------------------------------------
+// Event Name
+//
+// This is the name of the event displayed in the editor.
+//---------------------------------------------------------------------
+
+name: "Delete Bulk Messages MOD",
+
+//---------------------------------------------------------------------
+// Is Event
+//
+// Must be true for this to be an event.
+//---------------------------------------------------------------------
+
+isEvent: true,
+
+//---------------------------------------------------------------------
+// Event Variables
+//
+// The variables associated with this event. Can only have 0, 1, or 2.
+//---------------------------------------------------------------------
+
+fields: ["Temp Variable Name (stores list of messages):", "Temp Variable Name (stores amount of messages):"],
+
+//---------------------------------------------------------------------
+// Action Bot Mod
+//
+// Upon initialization of the bot, this code is run. Using the bot's
+// DBM namespace, one can add/modify existing functions if necessary.
+// In order to reduce conflictions between mods, be sure to alias
+// functions you wish to overwrite.
+//
+// This is absolutely necessary for custom event triggers since it
+// allows us to setup callbacks for the necessary events we would
+// like to be notified about.
+//
+// The client object can be retrieved from: `const bot = DBM.Bot.bot;`
+// Classes can be retrieved also using it: `const { Actions, Event } = DBM;`
+//---------------------------------------------------------------------
+
+mod: function(DBM) {
+
+	// Let's make our own namespace like the chads we are.
+	DBM.LeonZ = DBM.LeonZ || {};
+
+	// This function calls all events that use this trigger.
+	DBM.LeonZ.messageDeleteBulk = function(messagesList) {
+		// Grab them classes from the DBM namespace.
+		const { Bot, Actions } = DBM;
+
+		// Get all events that use this custom event trigger.
+		const events = Bot.$evts["Delete Bulk Messages MOD"];
+
+		// Ensure there are any.
+		// If the user did not create any events with this trigger, this will be null.
+		if(!events) return;
+
+		// Call each one.
+		const temp = {}
+		const server = messagesList.array()[0].guild;
+		for(let i = 0; i < events.length; i++) {
+			const event = events[i];
+			if(event.temp) temp[event.temp] = messagesList;

```

Comment:
perhaps storing `messagesList.array()` would be better since it'd be compatible with Get Item from List, etc?

Reply:
Done